hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -4155,6 +4155,12 @@ public CompactionState getCompactionState(final TableName tableName) {
           continue;
         }
         RegionMetrics regionMetrics = sl.getRegionMetrics().get(regionInfo.getRegionName());
+        if (regionMetrics == null) {
+          LOG.warn(String.format(
+            "Can not get compaction details for the region: %s , it may be not online.",
```

Comment:
log4j already has the parameterised feature, there's no need to use `String.format` here. Just do:

```
LOG.warn("Can not get compaction details for the region: {} , it may be not online.", regionInfo.getRegionNameAsString());
```

Reply:
The update has been made, thanks